### PR TITLE
Add troubleshooting expiring PIV cert guide

### PR DESCRIPTION
**Pull request by jmhooper**

### CODE REVIEW

File: _articles/toubleshooting-expiring-pivcac.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Using the cert:find_replacement rake task
+
+The `certs:find_replacement` rake task will make a best effort to find a
+replacement for a cert and save it for you.
+
+To run the rake task, invoke it like this:
+
+```shell
+# Replace the following with the expiring cert key ID:
+EXPORT expiring_key_id='CF:79:3C:ED:4D:BC:19:25:F2:45:69:4E:12:2F:9C:29:53:C9:A7:46'
```
Comment (zachmargolis) [inline _articles/toubleshooting-expiring-pivcac.md:45]: I've only seen bash EXPORT in lowers: `export`?
```suggestion
export expiring_key_id='CF:79:3C:ED:4D:BC:19:25:F2:45:69:4E:12:2F:9C:29:53:C9:A7:46'
```

**Review by zachmargolis — APPROVED** [review]
LGTM yay more PIV docs

File: _articles/toubleshooting-expiring-pivcac.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+---
+
+Whenever a certificate is due to expire, we should verify that a replacement is
+in place and remove the expiring cert.
+PIV issuers will usually issue a new certificate which will have a later
+expiration and probably a new subject key and subject key ID.
+
+The best place to look for a re-issued certificate is on the signing cert for
+that certificate.
+Within the PKI sphere, signing or parent cert will have a "Subject Information
```
Comment (mitchellhenke) [inline _articles/toubleshooting-expiring-pivcac.md:15]:
```suggestion
Within the PKI sphere, the signing or parent cert will have a "Subject Information
```

File: _articles/toubleshooting-expiring-pivcac.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+Finding replacement certificates is tricky because there are a lot of edge
+cases.
+A group that issues PIVs may change the structure of their branch of the PKI graph.
+Also, if the signing cert has been re-issued, and the replacement cert was issued
+with the new signing cert you may need to go further up the tree.
+
+Moreover, replacement certificates are often re-issued well before they expire,
+and PIVs are issued with the new certs.
+This means that often times the new certificate will already be in place.
+Be careful about adding duplicates.
```
Comment (mitchellhenke) [inline _articles/toubleshooting-expiring-pivcac.md:34]: do/can we have a test that checks for this?

File: _articles/toubleshooting-expiring-pivcac.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+```
+
+You can select the index of a cert to add it to the store.
+The test suite should validate that the certificate is valid and signed by a trusted root.
+You can run `bundle exec rspec spec/certs/store_spec.rb` to validate this.
+
+## Using the Subject Information Access extension
+
+It is possible to use openssl to perform the checks this rake task performs in
+you terminal.
```
Comment (mitchellhenke) [inline _articles/toubleshooting-expiring-pivcac.md:78]:
```suggestion
your terminal.
```

**Review by mitchellhenke — APPROVED** [review]
👍🏼 

Should we add a link to this page on https://handbook.login.gov/ ?

**Comment by zachmargolis** [comment]
> Should we add a link to this page on https://handbook.login.gov/ ?

The handbook index page automatically links to each article added

File: _articles/toubleshooting-expiring-pivcac.md (hunk 3 above)
Comment (jmhooper) [reply _articles/toubleshooting-expiring-pivcac.md:34]: We do indeed https://github.com/18F/identity-pki/blob/4d5f3577fd12152904cdce2a55a01d490de2bf68/spec/certs/store_spec.rb#L44

Comment (mitchellhenke) [reply _articles/toubleshooting-expiring-pivcac.md:34]: 👍🏼

**Pull request merged**